test/specs/utils.spec.js

```diff
@@ -0,0 +1,30 @@
+import utils from '../../src/utils'
+import {expect} from 'chai'
+
+const {checkMessageMarkdown} = utils
+
+describe('utils', () => {

```

Comment:
Would be nice a cypress integration test as well.

Reply:
Forsure, will work on that.

Reply:
@archfz  done